app/src/main/java/io/intrepid/contest/models/User.java

```diff
@@ -0,0 +1,96 @@
+package io.intrepid.contest.models;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+/*{
```

Comment:
We could create a standard JavaDoc "template" to use in our API models. Not sure if Retrofit has any standards.